sample_app/main_rtk.c

```diff
@@ -921,8 +146,8 @@ const shell_cmd_t cmd_pnio_run =
 {
       .cmd = _cmd_pnio_run,
       .name = "pnio_run",
-      .help_short = "Exec pnio",
-      .help_long ="Exec pnio"
+      .help_short = "Exec pnio (station name). Defaults to " APP_DEFAULT_STATION_NAME,
```

Comment:
The short description is used when listing all available commands and does not normally include arguments.

The long description is for detailed help and follows similar conventions to e.g. man. Suggest something like:

 .help_short = "Start pnio sample app",
 .help_long = 
    "pnio_run [station name]\n"
    "\n"
    "Start the pnio sample app. Station name defaults to " APP_DEFAULT_STATION_NAME " if not given.\n"

Reply:
Great! Now updated.